packages/mds-metrics/request-handlers.ts

```diff
@@ -0,0 +1,162 @@
+import log from '@mds-core/mds-logger'
+
+import db from '@mds-core/mds-db'
+import { inc, RuntimeError, yesterday } from '@mds-core/mds-utils'
+import { EVENT_STATUS_MAP } from '@mds-core/mds-types'
+import { now } from 'moment'
+import {
+  MetricsApiRequest,
+  MetricsApiResponse,
+  instantiateEventSnapshotResponse,
+  instantiateStateSnapshotResponse
+} from './types'
+
+export async function getStateSnapshot(req: MetricsApiRequest, res: MetricsApiResponse) {
+  const { params } = req
+  log.info(params)
+
+  const { start_time = yesterday(), end_time = now(), bin = 3600000 } = params
```

Comment:
These probably come from `req.query`, not `req.params` .. also, they should be typed.
```
export interface MetricsApiRequest extends ApiRequest {
  // Query string parameters always come in as strings
  query: Partial<
    {
      [P in 'start_time' | 'end_time' | 'bin']: string
    }
  >
}
```